core/src/client/client.rs

```diff
@@ -262,7 +262,7 @@ impl TimeoutHandler for Client {
             }
             RESEAL_MIN_TIMER_TOKEN => {
                 // Checking self.ready_parcels() for efficiency
-                if !self.ready_parcels().is_empty() {
+                if self.engine().engine_type() != EngineType::PBFT && !self.ready_parcels().is_empty() {
```

Comment:
Please add a method `use_reseal_min_timer` or `ignore_reseal_min_timer`

Reply:
I fixed it.